docs/reference/generated/accordion-panel.json

```diff
@@ -41,12 +40,10 @@
   },
   "cssVariables": {
     "--accordion-panel-height": {
-      "description": "The accordion panel's height.",
-      "type": "number"
+      "description": "The accordion panel's height."
```

Comment:
Can we keep the previous logic for the @type info?

Reply:
Yes, the extractor doesn't handle the @type tag yet, but will soon.